api_v3/services/UserEntryService.php

```diff
@@ -94,10 +95,6 @@ public function deleteAction($id)
 	 */
 	public function listAction(KalturaUserEntryFilter $filter, KalturaFilterPager $pager = null)
```

Comment:
But you are already forcing this in the filter itself so if they are not provided you throw an error